data/json/items/comestibles/dairy.json

```diff
@@ -7,7 +7,7 @@
     "copy-from": "milk",
     "spoils_in": "12 hours",
     "price": 19,
-    "parasites": 11,
+    "contamination": 30,
```

Comment:
30% is a very high contamination rate. When unpasteurized milk was the norm it was safe enough that a large majority of people didn't get poisoned by it.